src/SprykerEco/Service/CrefoPay/Dependency/Service/CrefoPayToUtilTextServiceBridge.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Service\CrefoPay\Dependency\Service;
+
+class CrefoPayToUtilTextServiceBridge implements CrefoPayToUtilTextServiceInterface
+{
+    /**
+     * @var \Spryker\Service\UtilText\UtilTextServiceInterface
+     */
+    protected $utilTextService;
+
+    /**
+     * @param \Spryker\Service\UtilText\UtilTextServiceInterface $utilTextService
+     */
+    public function __construct($utilTextService)
+    {
+        $this->utilTextService = $utilTextService;
+    }
+
+    /**
+     * @param int $length
+     *
+     * @return string
+     */
+    public function generateRandomString($length)
```

Comment:
please add return types for all bridges to cover the latest requirement from Mark

```
Your bridge should always be declared stricter than the parent interface (facade, client, ...) it delegates to. Make sure to set a return type as minimum to avoid arch issue/error thrown.
In general for your modules, make sure to set method return types wherever possible so that future extensions on it will have less impact then moving forward.
```